scripts/run_typedoc.sh

```diff
@@ -40,36 +43,67 @@ generate_docs() {
         echo -e "\033[0;93mGenerating typedocs\033[0m"
         pushd ../$2
         if [[ -z "$NOBUILD" ]]; then
+            git clean -xdf
             make ensure && make build && make install
```

Comment:
added this because i've had issues with some node module stuff not being pristine when doing thsi stuff.